bundles/org.openhab.binding.upb/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="upb" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="generic">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="serial-pim"/>
+		</supported-bridge-type-refs>
+		<label>Generic Powerline Device</label>
+		<description>A generic device in a UPB network</description>
+		<category>WallSwitch</category>
+
+		<channels>
+			<channel id="switch" typeId="switch"/>
```

Comment:
Because a dimmer channel also should accept on/off the convention we use it to not model a separate switch channel when using a dimmer. In items the dimmer channel can simply be configured as a switch. Therefor please remove this channel. Since the code already doesn't check for which channel the command is, the code already seems to support this use case.